### DISCUSSION TITLE
23.10.05 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#20] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - Spotify 로그인 후 사용자 정보 저장 로직 구현 (`name`, `email`, `provider_id`, `image`)
- 내일은 어떤 일을 하나요?
    - JWT 토큰 학습 및 구현 (access token & refresh token)
    - 패키지 구조 고민

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트 등록 컨트롤러 테스트 작성
    - 플레이리스트 컨트롤러 등록 구현
- 내일은 어떤 일을 하나요?
    - 플레이리스트 등록 서비스 테스트 작성
    - 플레이리스트 서비스 비즈니스 로직 구현

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 주문 도메인 생성
  * 주문하기 기능 추가(단 결제는 따로 구현 안함)
* 내일은 어떤 일을 하나요?
  * 주문 마저 완성
  * 테스트 DB 설정하기

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 노래 추가 외에도 다양한 알림에 대응할 수 있도록 기능 추가
  - 테스트 코드 작성(예외 부족)
- 내일은 어떤 일을 하나요?
  - 트랜잭션 세분화
  - 패키지 정리
